.circleci/config.yml

```diff
@@ -25,6 +28,12 @@ jobs:
           name: Execute ansible-lint
           command: >
             find . -type f -name '*.yml' | xargs ansible-lint -c /tmp/ansible_joke-programs_lint-rules/.ansible-lint-rules
```

Comment:
For these lint rules should we really be outsourcing it?

Reply:
I copied the ansible-lint rules directly into the template